packages/typespec-client-generator-core/src/decorators.ts

```diff
@@ -989,3 +1006,68 @@ export const $useSystemTextJsonConverter: DecoratorFunction = (
   entity: Model,
   scope?: LanguageScopes
 ) => {};
+
+const clientInitializationKey = createStateSymbol("clientInitialization");
+
+export const $clientInitialization: ClientInitializationDecorator = (
+  context: DecoratorContext,
+  target: Namespace | Interface,
+  options: Model,
+  scope?: LanguageScopes
+) => {
+  let isOg = isOperationGroupWithNoExplicitDecorator(target);
+  if (hasExplicitClientOrOperationGroup(context)) {
+    isOg = context.program.stateMap(operationGroupKey).get(target) !== undefined;
+  }
+  if (isOg) {
+    if (
+      target.namespace &&
+      context.program.stateMap(clientInitializationKey).get(target.namespace) &&
+      context.program.stateMap(clientInitializationKey).get(target.namespace)[AllScopes] === options
+    ) {
+      setScopedDecoratorData(context, $override, clientInitializationKey, target, options, scope);
+    } else {
+      reportDiagnostic(context.program, {
```

Comment:
for .net, they could support sub client with param, and they will generate such api for face. so, do we really need to add this check?

Reply:
let me go ahead and get rid of this check. This is explicitly for operation groups, not initializable subclients, but it is making the code more complicated than it needs to be, so let me roll this back